crates/bevy_scene/src/scene_spawner.rs

```diff
@@ -304,6 +306,18 @@ impl SceneSpawner {
             }
         }
     }
+
+    pub fn instance_is_ready(&self, instance_id: InstanceId) -> bool {
+        self.spawned_instances.contains_key(&instance_id)
+    }
+
+    pub fn for_entity_in_scene_instance(&self, instance_id: InstanceId, mut f: impl FnMut(Entity)) {
```

Comment:
Thoughts on removing `for_entity_in_scene_instance` in favor of:

```rust
fn iter_instance_entities(&self, instance_id: InstanceId) -> 
  Option<impl Iterator<Item=Entity>>  {
  self.spawned_instances
    .get(&instance_id)
    .map(|instance| instance.entity_map.values.cloned())
}
```

Reply:
I think it would make it a bit more flexible

Reply:
good idea! I tried using the same idea as [for_current_entity](https://docs.rs/bevy/0.4.0/bevy/ecs/struct.Commands.html#method.for_current_entity) but this seems better

Reply:
Yeah i'm not a huge fan of `for_current_entity`, but the builder pattern sort of forced my hand / i consider it to be an exception rather than a rule. SceneSpawner isn't a builder so I don't feel constrained in the same way.